src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/SpecimenQuantity.java

```diff
@@ -0,0 +1,58 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.Split;
+
+/**
+ * Example QTY+SVO:1750+:::mL'
+ */
+@RequiredArgsConstructor
+@Getter
+@EqualsAndHashCode(callSuper = false)
+public class SpecimenQuantity extends Segment {
+    private static final String KEY = "QTY";
+    private static final String QUALIFIER = "SVO";
+    private static final String KEY_QUALIFIER = KEY + PLUS_SEPARATOR + QUALIFIER;
+    private static final int THREE = 3;
+    private static final int FOUR = 4;
```

Comment:
Could we give this a more descriptive name?
```suggestion
    private static final int MEASUREMENT_UNIT_INDEX = 4;
```

Reply:
Sure! Thanks! f535ab469a9326e53ba885c0b39f74f59433aa09